lib/ansible/plugins/connection/network_cli.py

```diff
@@ -294,10 +300,12 @@ def send(self, command, prompt=None, answer=None, newline=True, sendonly=False):
         '''
         try:
             self._history.append(command)
+            display.debug("CLI-SEND: %s\\r" % self._escape_crnl(command))
```

Comment:
It is not recommended to log commands executed on the remote device due to security reasons.